tests/test_student_becomes_the_teacher.py

```diff
@@ -165,3 +166,50 @@ def test_get_class_average_negative(input_neg):
     except TypeError as e:
             assert str(e) == "{} Passed the wrong data type".format(input_neg)
 
+
+@pytest.mark.parametrize("test_input, expected", [
+    ([0], 0),
+    ([1], 1),
+    ([49], 49),
+    ([50], 50),
+    ([51], 51),
+    ([99], 99),
+    ([100], 100),
+    ([0, 0], 0),
+    ([0, 1], 0.5),
+    ([0, 49], 24.5),
+    ([25, 25], 25),
+    ([0, 51], 25.5),
+    ([99], 99),
+    ([100], 100),
+    ([0, 0, 0], 0.0),
+    ([0, 0, 1], 0.33),
+    ([0, 24, 25], 16.33),
+    ([0, 25, 25], 16.67),
+    ([0, 25, 26], 17.0),
+    ([33, 33, 33], 33.0),
+    ([33, 33, 34], 33.33)
+])
+def test_average_5(test_input, expected):
+    assert average_5(test_input) == expected
+
+
+@pytest.mark.parametrize("input_negative", [
+    None,
+    True,
+    False,
+    0,
+    1,
+    "",
+    "Test row",
+    {},
+    (),
+    ["B", 0]
+])
+def test_average_5_negative(input_negative):
+    try:
+        average_5(input_negative)
```

Comment:
you still need to add a `raise Exception("blah")` or `pytets.fail("blah")` after this line to fail test if there was no exception thrown from the average_5() call